hack/buildkitd_build_push_image.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+
+# Run BuildKit in the background
+echo "Starting BuildKit daemon in the background..."
```

Comment:
1. The service needs to stop after the image was created and pushed. 
2. Create a "Make" task  to clean up the environment (prune cache, images, and so on)